CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+### 0.10.0
```

Comment:
This is a good start, but we're going to need a lot more here before we release. There are deprecation notices, compatibility notes, and other things we need to highlight here.

Also should include updates to the docs/ dir and a new release notes page.